content/en/docs/Core concepts/metrics.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Metrics"
+date: 2020-11-03
+weight: 2
+description: >
+  How to metrics from Baker
+---
+
+### Metrics
+
+ During execution, Baker collects different kind of performance data points:
+
+ - general pipeline metrics such as the total number of records processed, records per seconds.
+ - component-specific metrics: files written per second, discarded records (in a filter), errors, etc.
+ - Go runtime metrics, mallocs, frees, garbage collections and so on.
+
+If enabled, Baker collects all these metrics and forwards them to a metrics client.
+
+When configured, a Baker topology also configures the metrics client to use, if any.
+Metrics configuration is performed in Baker configuration TOML file.
```

Comment:
many repetitions of config-something.
and I think these 3 sentences are clear enough, I have the feeling that something is missing to give them the right weight

Reply:
if not described in other parts of the doc, I'd add how to disable the metrics